source/ed/osm2ed.cpp

```diff
@@ -674,13 +674,7 @@ void PoiHouseNumberVisitor::node_callback(uint64_t osm_id, double lon, double la
     this->fill_housenumber(osm_id, tags, lon, lat);
     auto logger = log4cplus::Logger::getInstance("log");
```

Comment:
This line is now useless, and needed in `insert_data`. The compilation fails because of that.